src/intersection-observer.js

```diff
@@ -208,6 +209,10 @@ export class IntersectionObserver extends Observable {
    * @private
    */
   flush_() {
+    if (!this.isElementInDoc_()) {
```

Comment:
@jridgewell Can you confirm on this? Helps me to decide if I want to add that safety check here?